src/main/kotlin/no/nav/syfo/App.kt

```diff
@@ -39,8 +39,11 @@ fun main() {
         )
     )
 
+    val personFlagget84AivenProducer = createPersonFlagget84AivenProducer(environment)
+    val personFlagget84AivenConsumer = createPersonFlagget84AivenConsumer(environment)
```

Comment:
Domenespørsmål: Hva er egentlig en personflagget84? 🤓

Reply:
Ja, si det. Vet du kanskje @JMLindseth?

Reply:
Det kommer av at vi rotet litt med navngivning 😬 Tanken var at vi flagger en person (for at de ikke skal behandles automatisk) på bakgrunn av §8-4 i folketrygdloven som handler om når man har rett på sykepenger (bl.a. at man skal være i aktivitet).